From fb2cd1dd271eb233e14c254bdcc429153bbbf945 Mon Sep 17 00:00:00 2001
From: "Haijun.Zhang" <Haijun.Zhang@freescale.com>
Date: Wed, 15 May 2013 08:30:02 +0800
Subject: [PATCH 166/987] Powerpc/dts: Add zarlink device node support for
 p1020rdb-pd

[Original patch taken from QorIQ-SDK-V1.6-SOURCE-20140619-yocto.iso]

On P1020RDB-PD board SPI interface is connected to the TDM chips SLIC/SLAC.
SPI_CS1_N -->> Zarlink Le88266DLC
SPI_CS2_N -->> Zarlink Le88266DLC
Currently the p1020rdb-pd.dtsi missed to add this two nodes. Without these
nodes Le88266DLC will not be probed correctly and TDM will not work.
This should be support as default.

Signed-off-by: Haijun Zhang <haijun.zhang@freescale.com>
Change-Id: I9add25537f05dd09a87d59416f5e053d91c3e067
Reviewed-on: http://git.am.freescale.net:8181/2495
Reviewed-by: Xie Xiaobo-R63061 <X.Xie@freescale.com>
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>
Tested-by: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>
---
 arch/powerpc/boot/dts/p1020rdb-pd.dtsi | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/arch/powerpc/boot/dts/p1020rdb-pd.dtsi b/arch/powerpc/boot/dts/p1020rdb-pd.dtsi
index 02e9358..6e07fa7 100644
--- a/arch/powerpc/boot/dts/p1020rdb-pd.dtsi
+++ b/arch/powerpc/boot/dts/p1020rdb-pd.dtsi
@@ -186,6 +186,17 @@
 				label = "file system jffs2";
 			};
 		};
+		slic@0 {
+			compatible = "zarlink,le88266";
+			reg = <1>;
+			spi-max-frequency = <8000000>;
+		};
+		slic@1 {
+			compatible = "zarlink,le88266";
+			reg = <2>;
+			spi-max-frequency = <8000000>;
+		};
+
 	};
 
 	mdio@24000 {
-- 
1.9.1

